packages/dashboard/src/components/actions/index.tsx

```diff
@@ -45,6 +48,11 @@ const Actions: React.FC<ActionsProps> = ({
       ...dashboardConfiguration,
       viewport: viewport ?? DEFAULT_VIEWPORT,
     });
+
+    metricsRecorder?.record({
+      metricName: 'DashboardSave',
+      metricValue: 1,
+    });
   };
```

Comment:
Why do we do this in the here and not in iot-application?

Reply:
@tracy-french we should track both 👍 
@jmbuss the implementation of the `metricsRecorder` is the consumers' responsibility; if nothing is provided to `registerPlugin('metricsRecorder', ...)`, the `metricsRecorder` is undefined; otherwise, the consumer needs to make sure the provided `metricsRecorder` works